src/v2/models/AppState.js

```diff
@@ -53,8 +53,15 @@ export default Model.extend({
     showSignoutLink: {
       deps: ['currentState'],
       fn: function (currentState = {}) {
-        // hide signout for IDENTIFY step
-        return _.isFunction(currentState.cancel) && currentState.step !== 'IDENTIFY';
+        // hide signout for IDENTIFY and ENROLL step
+        if (_.isFunction(currentState.cancel)) {
+          let showSignout = true;
+          if (currentState.step !== 'IDENTIFY'||
+            currentState.step !== 'ENROLL') {
```

Comment:
why hide sign-out in `enroll` step?

Reply:
sign out in enroll step does not make sense, since the user has not logged in and just hit sign up button from login screen